.github/workflows/ci.yml

```diff
@@ -155,8 +155,6 @@ jobs:
         uses: foundry-rs/foundry-toolchain@v1
         with:
           version: nightly
-      - name: Run Forge tests
-        run: yarn workspace @balancer-labs/v2-pool-stable test-fuzz
```

Comment:
I'd leave it just to be consistent. Otherwise we might want to delete it elsewhere if it's not used (e.g. linear pools).